src/blueprints/tv2/timeline-state-resolver-types/tv2-caspar-cg-types.ts

```diff
@@ -14,7 +14,7 @@ export interface Tv2CasparCgTemplateData {
       display: Tv2CasparCgTemplateDisplayMode
       payload: {
         type: string
-        0: string
+        [locatorIndex: number]: string
```

Comment:
I see what you did here, but I do have a slight concern that since this is the type used by TSR we should keep it "identical" to TSR as well. And in TSR the type is `0: string, 1: string`.
I know this is a bad type, but with your suggestion one could think that `2: string, 3: string` etc also are valid options.

Reply:
In the HTML graphics project that we use there is a lower third (Headline) that uses 0, 1 and 2. If we want to have the specific entries, we can do something like:
```typescript
type LowerThirdLocators =
  | { 0: string }
  | { 0: string, 1: string }
  | { 0: string, 1: string, 2: string }
```
or add all supported graphic templates from the project to the types. Personally, I think both will be a bit too cumbersome to manage – unless we decide that we want to validate the data before sending it to CasparCG.

Reply:
That's also not great. I am okay with your current solution. I just wanted to raise a little flag.